app/controllers/users_controller.rb

```diff
@@ -74,6 +74,7 @@ def index
   def show
     # show projects based on current user visibility
     @memberships = @user.memberships
+                        .where.not(project_id: nil)
```

Comment:
using the `.global` scope is more descriptive. 

I'll add the necessary change.

Reply:
Sorry, read it the wrong way around